MIP108/MIP108.md

```diff
@@ -233,9 +233,9 @@ The new brand must be launched alongside the launch of the NewStable and NewGovT
 
 ###### 2.1.2.3.1
 
-When the new brand and associated website have been fully developed and are ready for launch, the Accessibility Facilitator must publish the relevant materials related to the new brand, and then trigger a governance poll following the Weekly Governance Cycle to ratify or reject the new brand.
+When the new brand and associated website have been fully developed and are ready for launch, the Accessibility Facilitator must directly take all necessary steps to implement the new brand and its related launch marketing campaigns, in a single, coordinated effort. The new products must be launched simultaneously or following the launch of the new brand, in order to maximize market impact and success. All Facilitators must support in triggering whatever type of needed executive vote, or other type of vote, to support the impactful and seamless launch of the Endgame Products and brand.
 
-###### 2.1.2.3.1.1: If accepted, this governance poll will also instantly edit all Alignment Artifacts to modify the words used to be in line with the new brand. No changes to the logic of the Alignment Artifacts must be made, other than names and brand related descriptions and addition of brand-related descriptions. The rewritten Alignment Artifacts must be submitted alongside the other brand materials. This can also apply to the language of a proposal that is in the MIP102 process.
+###### ACC2.1.2.3.1.1: Following the reveal of the new brand, the MIP portal must be prepared for the vote to upgrade the Atlas to the Atlas v2 version, using a weekly governance poll. This must happen after the implementation of the new brand, and will involve the full adoption of the new brand in the Atlas. Until the upgrade to Atlas v2, the existing Atlas will continue to refer to the existing brand, temporarily, while all external facing websites and frontends must refer to the new branding
```

Comment:
This is not the current numbering convention. We suggest reverting to the standard currently used. If we wish to update the numbering convention that should be done separately and applied to all articles.

```suggestion
###### 2.1.2.3.1.1: Following the reveal of the new brand, the MIP portal must be prepared for the vote to upgrade the Atlas to the Atlas v2 version, using a weekly governance poll. This must happen after the implementation of the new brand, and will involve the full adoption of the new brand in the Atlas. Until the upgrade to Atlas v2, the existing Atlas will continue to refer to the existing brand, temporarily, while all external facing websites and frontends must refer to the new branding
```